tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -322,6 +323,16 @@ function kbs_k8s_deploy() {
 			# ITA/ITTS specific configuration
 			sed -i -e "s/tBfd5kKX2x9ahbodKV1.../${ITA_KEY}/g" kbs-config.toml
 		popd
+		
+		# Ideally this should be something kustomizable on trustee side.
+		#
+		# However, for now let's take the bullet and do it here, and revert this as
+		# soon as https://github.com/confidential-containers/trustee/issues/567 is
+		# solved.
+		pushd "${COCO_KBS_DIR}/config/kubernetes/base/"
+			sed -i "s|^\(\s*\)volumes:|\1  env:\n\1    - name: https_proxy\n\1      value: \"${HTTPS_PROXY}\"\n\1volumes:|" deployment.yaml
```

Comment:
I definitely don't love the `sed` to inject based on the `volumes`, which I guess is to add this `env` section into the kbs containers spec, as it feels quite brittle, but if it's short-term that should alleviate some of that concern.

Reply:
I hate the `sed`, I sincerely do. :-)
We could yq that, but that's as fragile as the sed.

Reply:
Change to use yq instead.

Reply:
An issue with `yq` is that it's yaml formatting doesn't match ours, but that's not a big problem when we are doing a modification in the CI and not checking any of it in